### PR TITLE
[snap] add libssl1.1 from bionic for Qt's consumption

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

**Comment by Saviq** [comment]
```
multipass launch https://cdimage.debian.org/mirror/cdimage/openstack/testing/debian-testing-openstack-amd64.qcow2 --name https-debian                                                     
Launched: https-debian
```

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -126,6 +126,15 @@ parts:
...
     source-type: deb
 
+  libssl1.1:
+    plugin: dump
+    source:
+    - on amd64: http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4.3_amd64.deb
+    - on i386: http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4.3_i386.deb
+    - on armhf: http://ports.ubuntu.com/ubuntu-ports/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4_arm64.deb
+    - on arm64: http://ports.ubuntu.com/ubuntu-ports/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4_armhf.deb
+    source-type: deb
```
Comment (ricab) [inline snap/snapcraft.yaml]: Last two architectures swapped (just noticed)

Comment (ricab) [reply snap/snapcraft.yaml]: Thanks for the fix. Why is '.3' missing after 4?

Comment (ricab) [reply snap/snapcraft.yaml]: thanks

**Pull request merged**